.github/workflows/publish-docker.yml

```diff
@@ -0,0 +1,20 @@
+name: Publish Docker
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  publish-docker:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: docker/build-push-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
```

Comment:
@kbariotis is this GitHub repo configured to provide these secrets?

Reply:
No but I should be able to set these up soon. :)